src/Installers/Windows/SharedFrameworkBundle/thm.xml

```diff
@@ -58,9 +58,9 @@
     </Page>
     <Page Name="Success">
         <Text Name="SuccessHeader" X="11" Y="80" Width="-11" Height="30" FontId="2" HideWhenDisabled="yes" DisablePrefix="yes">#(loc.SuccessHeader)</Text>
```

Comment:
Maybe bump this one to 34 as well? It's the same page displayed by the wizard

Reply:
done

Reply:
Wait, the error is coming from the hosting bundle, not the shared framework bundle. I tried the string in the framework bundle and it already fits.

![image](https://user-images.githubusercontent.com/7409981/100948894-b8462480-34bd-11eb-9074-73007307a602.png)

The hosting bundle has a narrow layout (windows is less wide), which is why the string is being trimmed.

Reply:
Oh, whoops - fixed